doc/manuals/admin/build_source.md

```diff
@@ -1,6 +1,6 @@
 # Building from sources
 
-Orion Context Broker reference distribution is Debian 12. This doesn't mean that the broker cannot be built in other distributions (actually, it can). This section also includes indications on how to build in other distributions, just in the case it may help people that don't use Debian. However, note that the only "officially supported" procedure is the one for Debian 12; the others are provided "as is" and can get obsolete from time to time.
```

Comment:
Although after the removal of the Ubuntu section the only procedure would be the one about Debian 12, I'd suggest to keep this fragment just in case in the future another non official procedures would be included, with the following change:

> This section also includes indications...

to

> This section may include indications...

Reply:
I updated it.